src/libaktualizr/package_manager/dockerapp_standalone.cc

```diff
@@ -168,6 +168,7 @@ data::InstallationResult DockerAppStandalone::install(const Uptane::Target &targ
     updateNotify();
     res = OstreeManager::install(target);
     if (res.result_code.num_code == data::ResultCode::Numeric::kInstallFailed) {
+      // TODO: num_code is supposed to be kOk or kNeedCompletion, otherwise it's considered as failed?
```

Comment:
I think, we shouldn't make an assumption about a function implementation and what it can return or not, we have a function contract/signature and should be able to handle properly any value it returns according to this contract. It means that the code should handle any possible value of res.result_code.num_code.
Having said that, I think it doesn't make sense to change anything here taking into account that the given implementation is going to be decommissioned in favor of the docker-compose apps (https://github.com/doanac/aktualizr-lite/tree/compose-apps).